### ISSUE TITLE
device_tracker.x reports exact address instead of home/not_home

### ISSUE THREAD

**Issue by bepstein111** [gcobb321/icloud3#74]
This is more of a standardization issue I think, or I might be missing something entirely, but every other device tracker integration for home assistant I've seen (including original iCloud) returns either home or not_home for State with it's device_tracker.x entities. When I have xfinity (router) and iCloud3 enabled as trackers simultaneously (maybe a bad idea?) person.x will switch constantly between "home" and my home address. I know there's the sensor.x_badge entity, but you can't use that as a person's device tracker.

tl;dr I want my person.x entities to report either home or not_home, instead of not_home or my home address is this possible?

I'm happy to explain more/provide configs if this was confusing or didn't make a ton of sense.

Fantastic job on this btw, this seems really well done.

**Comment by gcobb321** [comment]
iCloud3 does not change the state to the address, it only updates the state to home or not_home so the address must be coming from another tracker. I would suggest using the special 'zone' sensor that is only updated by iCloud3. While the badge shows the Zone name or distance from Home, the sensor.devicename_zone (and sensor.devicename_zone_name_1) report home, away or the zone name. Read more about [Zone Sensors here](https://gcobb321.github.io/icloud3_dev/#/chapters/2-sensors?id=zone-sensors).

**Comment by bepstein111** [comment]
Very interesting. I think it ended up being that my Home Assistant instance was named with my street address and not just "Home". Anyway, thanks for the info, it ended up pointing me in the right direction.
